services/worker-manager/test/provisioner_test.js

```diff
@@ -48,7 +48,7 @@ helper.secrets.mockSuite(testing.suiteName(), ['azure'], function(mock, skipping
                 msg => msg.Type === 'test-provision' && msg.Fields.workerPoolId === wt.workerPoolId), {
                 Logger: `taskcluster.worker-manager.provider.${pId}`,
                 Type: 'test-provision',
-                Fields: {workerPoolId: wt.workerPoolId, existingCapacity: wt.existingCapacity},
+                Fields: {workerPoolId: wt.workerPoolId},
```

Comment:
This message is coming from here:
https://github.com/taskcluster/taskcluster/blob/3d4c26818fbdf5b2322f2a1286146a359f0b76d9/services/worker-manager/src/providers/testing.js#L29-L31
so you'll want to update that as well.